docs/Scaling compute/Polly CLI.md

```diff
@@ -236,17 +321,19 @@ While creating a docker to be run on Polly, the following must be taken care of.
 
 *   Only self contained dockers can be run on Polly. A self contained docker is one which has the code to get input files as well as upload output files back contained in the docker.
 
-*   Public as well as Private dockers are supported. In order to run Private dockers, “secret” should be passed as a key in the json file. To get the secret key for the private docker, the following steps need to be followed.
+*   Public as well as Private dockers are supported. In order to run Private dockers, “secret” should be passed as a key in the json file. If your private dockers are on Polly itself , you don't require to generate this secret.
```

Comment:
Replace with "Public as well as private dockers are supported. In order to run private dockers, “secret” should be passed as a key in the .json file. If your private dockers are on Polly itself, you don't require to generate this secret."